src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+
+import org.example.consumer.UserConsumer;
+import org.example.producer.UserProducer;
+
+public class Main {
+    public static void main(String[] args) {
+        String topic = "user-topic";
+        String schemaRegistry = "http://localhost:8081";
+        String kafkaServer = "localhost:9092";
```

Comment:
Optional: create properties file, place configuration over there and read that file in the main method.